grafana/dashboard.json

```diff
@@ -331,13 +276,12 @@
         }
       ],
       "title": "DHT nodes",
-      "transformations": [],
       "type": "stat"
     },
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "prometheus"
```

Comment:
You should use `${DS_PROMETHEUS}` and not a specific UID that may not exist for other users